.gitignore

```diff
@@ -59,30 +59,6 @@ crashlytics-build.properties
 .gradle
 build/
 
-
-### Maven template
-target/
-pom.xml.tag
-pom.xml.releaseBackup
-pom.xml.versionsBackup
-pom.xml.next
-release.properties
-dependency-reduced-pom.xml
-buildNumber.properties
-.mvn/timing.properties
-### Eclipse template
-*.pydevproject
-.metadata
-bin/
-tmp/
-*.tmp
-*.bak
-*.swp
-*~.nib
-local.properties
-.settings/
-.loadpath
-

```

Comment:
I think the Eclipse template should be left in, I personally don't use Eclipse much but I think it'd be nice to keep in case someone contributing does use it

Reply:
It's a mistake on my part, I just wanted to remove for Maven...